app/dashboard/views.py

```diff
@@ -193,6 +194,63 @@ def remove_interest(request, bounty_id):
 
     return JsonResponse({'success': True})
 
+@require_POST
+@csrf_exempt
+def uninterested(request, bounty_id, profile_id):
```

Comment:
At the moment I have no idea how to test views, can't find any view tests in repo. Should I create new from the scratch?

Reply:
@mbeacom might know. i do not

Reply:
@kziemianek You can hold off on the view tests. I'm going to be pushing up some view tests along with the django auth branch that'll make it easier for others to follow the pattern moving forward. I'll add a test for this view following that and ping you, so you can check it out.